pxd.c

```diff
@@ -433,7 +433,7 @@ int pxd_handle_device_limits(struct fuse_req *req, uint32_t *size, uint64_t *off
 	max_sectors = blk_queue_get_max_sectors(q, op);
 	while (rq_sectors > max_sectors) {
 		struct bio *b;
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,18,0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,18,0) || (LINUX_VERSION_CODE >= KERNEL_VERSION(4,12,0) && defined(bvec_iter_sectors))
```

Comment:
Too many repeats, should try and move to pxd_compat.h

Reply:
Are you suggesting we move this check `(LINUX_VERSION_CODE >= KERNEL_VERSION(4,12,0) && defined(bvec_iter_sectors))` to a separate define?